examples/wordcount/docker/wordcount/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.6
+ADD . /code
+
+RUN apt-get update
+RUN echo "/usr/local/lib" >> /etc/ld.so.conf
+RUN git clone https://github.com/edenhill/librdkafka.git /tmp/librdkafka
+RUN ls /tmp/ && cd /tmp/librdkafka && ./configure && make && make install && ldconfig
+
+WORKDIR /code/examples/wordcount/
+RUN pip --version
+#RUN pip install -e git+https://github.com/confluentinc/confluent-kafka-python.git#egg=confluent-kafka
```

Comment:
I think the reason this does not get pulled in automatically for us might be a quirk in our local setup @ah-